README.md

```diff
@@ -46,11 +46,9 @@ Follow the instructions under the topic -  "Run the node"
 
 ```
 
-$ yarn demo
+$ npx tsx demo/src/func-test.ts
 
-$ yarn demo-vc
-
-$ yarn bench
+$ npx tsx demo/src/network-score-test.ts
```

Comment:
Failing with error 
(Not sure if this change has anything to do with it, just highlighting here)
    
    
/Users/niranjanp/DW/source/npci-build/cord.js/packages/chain-space/src/ChainSpace.chain.ts:473
    throw new SDKErrors.CordDispatchError(
          ^


CordDispatchError: Error dispatching delegate authorization: [object Object]
    at Object.dispatchDelegateAuthorization (/Users/niranjanp/DW/source/npci-build/cord.js/packages/chain-space/src/ChainSpace.chain.ts:473:11)
    at main (/Users/niranjanp/DW/source/npci-build/cord.js/demo/src/network-score-test.ts:107:24)

Reply:
would like to check this.

Reply:
the error is happening on 'develop' branch, which will be tested further. But this will get in for release branch now.